src/components-examples/material/table/table-http/table-http-example.ts

```diff
@@ -57,7 +54,12 @@ export class TableHttpExample implements AfterViewInit {
           this.resultsLength = data.total_count;
           return data.items;
         })
-      ).subscribe(data => this.data = data);
+      );
+    this._changeDetectorRef.detectChanges();
```

Comment:
Are we sure that this isn't masking the root cause of the error? Maybe the root cause is the `startWith` above which is updating the view immediately in `ngAfterViewInit` which is what usually causes "changed after checked" errors.

Reply:
I can test that out some more, but I don't think that the error is due to the emission of the Observable, but instead the assignment of the Observable to the field, i.e. the field going from `undefined` to `Observable<GithubIssue[]>`.

Reply:
If we go this route we'd need some good commenting on why this is necessary. It doesn't immediately seem obvious that this is a better approach than having a `subscribe`